src/AlgebraicGeometry/ToricVarieties/Proj/constructors.jl

```diff
@@ -1,10 +1,11 @@
 @doc raw"""
-    function proj(E::ToricLineBundle...)
+	proj(E::ToricLineBundle...)
```

Comment:
```suggestion
    proj(E::ToricLineBundle...)
```
Please use spaces instead of tabs for indentation, see https://docs.oscar-system.org/dev/DeveloperDocumentation/styleguide/#Whitespace